packages/paste-core/layout/stack/src/index.tsx

```diff
@@ -81,37 +76,42 @@ export const getStackChildMargins = (orientation: StackOrientation, spacing: Spa
   return styles;
 };
 
-const Stack = React.forwardRef<HTMLDivElement, StackProps>(({children, orientation, spacing, ...props}, ref) => {
-  const [childrenCount, validChildren] = React.useMemo(() => {
-    const filteredChildren = React.Children.toArray(children).filter(
-      (child) => React.isValidElement(child) || typeof child === 'string'
+const Stack = React.forwardRef<HTMLDivElement, StackProps>(
+  ({children, orientation, spacing, element = 'STACK', ...props}, ref) => {
+    const [childrenCount, validChildren] = React.useMemo(() => {
+      const filteredChildren = React.Children.toArray(children).filter(
+        (child) => React.isValidElement(child) || typeof child === 'string'
+      );
+      return [filteredChildren.length, filteredChildren];
+    }, [children]);
+    const stackStyles = React.useMemo(() => getStackStyles(orientation), [orientation]);
+    const childMargins = React.useMemo(() => getStackChildMargins(orientation, spacing), [orientation, spacing]);
+    const keySeed = useUIDSeed();
+
+    return (
+      <Box element={element} {...safelySpreadBoxProps(props)} {...stackStyles} ref={ref}>
+        {validChildren.map((child, index) => {
+          return (
+            <Box
+              element={`${element}_CHILD`}
+              {...(childrenCount !== index + 1 ? childMargins : null)}
+              key={keySeed(`stack-${index}`)}
+            >
+              {child}
+            </Box>
+          );
+        })}
+      </Box>
     );
-    return [filteredChildren.length, filteredChildren];
-  }, [children]);
-  const stackStyles = React.useMemo(() => getStackStyles(orientation), [orientation]);
-  const childMargins = React.useMemo(() => getStackChildMargins(orientation, spacing), [orientation, spacing]);
-  const keySeed = useUIDSeed();
-
-  return (
-    <Box {...safelySpreadBoxProps(props)} {...stackStyles} ref={ref}>
-      {validChildren.map((child, index) => {
-        return (
-          <Box {...(childrenCount !== index + 1 ? childMargins : null)} key={keySeed(`stack-${index}`)}>
-            {child}
-          </Box>
-        );
-      })}
-    </Box>
-  );
-});
+  }
+);
 
 Stack.displayName = 'Stack';
 
-if (process.env.NODE_ENV === 'development') {
-  Stack.propTypes = {
-    orientation: ResponsiveProp(PropTypes.oneOf(['horizontal', 'vertical'])).isRequired,
-    spacing: isSpaceTokenProp,
-  };
-}
+Stack.propTypes = {
+  orientation: ResponsiveProp(PropTypes.oneOf(['horizontal', 'vertical'])).isRequired,
+  spacing: isSpaceTokenProp,
+  element: PropTypes.string,
```

Comment:
Question here, looks like `element` wasn't added to Media Object's `propTypes` in that PR. Should it have been, or is this unnecessary?

Reply:
Yes, it should have